scripts/asm-installer/install_asm

```diff
@@ -2083,15 +2083,8 @@ is_membership_crd_installed() {
 populate_environ_info() {
   if [[ -n "${ENVIRON_PROJECT_ID}" && -n "${HUB_MEMBERSHIP_ID}" ]]; then return; fi
   if ! is_membership_crd_installed; then return; fi
-  local IDENTITY_PROVIDER
-  local IDENTITY
-  IDENTITY_PROVIDER="$(kubectl get memberships.hub.gke.io membership -o=json | jq .spec.identity_provider)"
-  IDENTITY="$(echo "${IDENTITY_PROVIDER}" | sed 's/^\"https:\/\/gkehub.googleapis.com\/projects\/\(.*\)\/locations\/global\/memberships\/\(.*\)\"$/\1 \2/g')"
-  if [[ -n "${IDENTITY}" ]]; then
-    read -r ENVIRON_PROJECT_ID HUB_MEMBERSHIP_ID <<EOF
-${IDENTITY}
-EOF
-  fi
+  HUB_MEMBERSHIP_ID="$(kubectl get memberships.hub.gke.io membership -o=json | jq .spec.owner.id | sed 's/^\"\/\/gkehub.googleapis.com\/projects\/\(.*\)\/locations\/global\/memberships\/\(.*\)\"$/\2/g')"
```

Comment:
I think this might also be "container.googleapis.com" not just "gkehub.googleapis.com"

Reply:
Have an example in the comment below. Looks like currently it is always the same "gkehub.googleapis.com"

Reply:
This is confirmed. gkehub resource uri is the owner.